python/lsst/dax/ppdb/sql/_ppdb_sql.py

```diff
@@ -544,6 +566,13 @@ def _store_objects(
 
         table = self._get_table("DiaObject")
 
+        if self._use_mjd_tai:
```

Comment:
Same observation as above - getting this information from an APDB interface would be a better solution than hard-coding field names here.